lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -294,20 +287,10 @@ sub engine_workit ($job, $callback) {
     log_info('+++ setup notes +++', channels => 'autoinst');
     log_info(sprintf("Running on $hostname:%d ($sysname $release $version $machine)", $instance),
         channels => 'autoinst');
-
     log_error('Failed enabling subreaper mode', channels => 'autoinst') unless session->subreaper;
-
-    # XXX: this should come from the worker table. Only included
-    # here for convenience when looking at the pool of
-    # debugging.
     my $job_settings = $job_info->{settings};
-    for my $i (qw(QEMUPORT VNC OPENQA_HOSTNAME)) {
-        $job_settings->{$i} = $ENV{$i};
-    }
-    if (open(my $fh, '>', 'job.json')) {
-        print $fh Cpanel::JSON::XS->new->pretty(1)->encode($job_info);
-        close $fh;
-    }
+    $job_settings->{$_} = $ENV{$_} for qw(QEMUPORT VNC OPENQA_HOSTNAME);
+    path('job.json')->spew(Cpanel::JSON::XS->new->pretty(1)->encode($job_info));
```

Comment:
Before this was silently ignoring when the file could not be opened, now it will die.
But I guess it's ok.